src/i18n.js

```diff
@@ -192,6 +220,10 @@ const resources = {
       'desc.source': 'вихідний код',
       'wifi.login': 'Вхід в WiFi',
       'wifi.name': 'Назва мережі',
+      'wifi.size': 'Розмір',
```

Comment:
@Teraskull is it grammatically correct?

Reply:
Yes

Reply:
Thanks. I marked wrong and I mean all lines: from 223 to 226.

Reply:
Yep, here everything is good.

Reply:
Thanks!